working/templates/SharedItems/SkylineDevopsHelp/GettingStarted.md

```diff
@@ -62,70 +64,77 @@ You can publish your artifact either manually via the Visual Studio IDE or by se
+- By adjusting the package version property:
 
-1. Search for Package Version.
+  1. Navigate to your project in Visual Studio, right-click, and select *Properties*.
 
-1. Adjust the value as needed.
+  1. Search for *Package Version*.
 
-### Changing the Version - Alternative
+  1. Adjust the value as needed.
 
-1. Navigate to your project in Visual Studio and double-click it.
+- By adjusting the *Version* XML tag:
 
-1. Adjust the "Version" XML tag to the version you want to register.
+  1. Navigate to your project in Visual Studio and double-click it.
 
-   ```xml
-   <Version>1.0.1</Version>
-   ```
+  1. Adjust the *Version* XML tag to the version you want to register.
+
+     ```xml
+     <Version>1.0.1</Version>
+     ```
 
 <!--#elseif (IsCatalogBasicCICD)-->
-## Publishing to the Catalog with Basic CI/CD Workflow
+## Publishing to the Catalog with basic CI/CD workflow
+
+This project includes a basic GitHub workflow for Catalog publishing.
 
-This project includes a basic GitHub workflow for Catalog publishing.  
 Follow these steps to set it up:
 
-1. Create a GitHub repository by going to **Git > Create Git Repository**, selecting GitHub, and filling in the wizard before clicking **Create and Push**.
+1. Create a GitHub repository by going to *Git* > *Create Git Repository* in Visual Studio, selecting GitHub, and filling in the wizard before clicking *Create and Push*.
 
 1. In GitHub, go to the *Actions* tab.
 
 1. Click the workflow run that failed (usually called *Add project files*).
 
-1. Click the "build" step that failed and read the failing error.
+1. Click the "build" step that failed and read the error.
 
    ``` text
    Error: DATAMINER_TOKEN is not set. Release not possible!
    Please create or re-use an admin.dataminer.services token by visiting: https://admin.dataminer.services/.
-   Navigate to the right Organization then go to Keys and create/find a key with permissions to Register Catalog Items.
+   Navigate to the right Organization then go to Keys and create/find a key with permissions to Register catalog items, Download catalog versions and Read catalog items.
    Copy the value of the token.
    Then set a DATAMINER_TOKEN secret in your repository settings: **Dynamic Link**
    ```
 
    You can use the links from the actual error to better address the next couple of steps.
 
-1. Obtain an **Organization Key** from [admin.dataminer.services](https://admin.dataminer.services/) with the following scopes:
-   - **Register catalog items**
-   - **Read catalog items**
-   - **Download catalog versions**
+1. Obtain an **organization key** from [admin.dataminer.services](https://admin.dataminer.services/) with the following scopes:
 
-1. Add the key as a secret in your GitHub repository, by navigating to **Settings > Secrets and variables > Actions** and creating a secret named `DATAMINER_TOKEN`.
+   - *Register Catalog items*
+   - *Read Catalog items*
```

Comment:
Since in the UI "catalog" isn't capitalized here, we'd better not capitalize it for the sake of consistency:
```suggestion
   - *Register catalog items*
   - *Read catalog items*
```